assets/less/formstyles.less

```diff
@@ -78,25 +78,52 @@ button:focus {
     }
   }
 }
-.btn-primary {
-  background: #2bb673;
-  color: #fff;
-}
-.btn.btn-danger {
-  background-color: #f00;
-  color: #fff;
+.btn-primary, .btn-success {
+  color: @buttonText;
+  background-color: @primaryGreen;
+  border-color: @primaryGreen;
+  &:disabled, &[disabled]:hover {
+    background-color: @primaryGreenDisabled;
+    border-color: @primaryGreenDisabled;
+    color: @buttonTextDisabled;
+  }
 }
+
 .btn-secondary {
-  background-color: #0082BF;
-  color: #fff;
+  color: @buttonText;
+  background-color: @secondaryBlue;
+  border-color: @secondaryBlue;
+  &:disabled, &[disabled] {
```

Comment:
This looks a bit redundant without the `:hover`.  Also consider my suggestion in the previous comment.